discord/user.py

```diff
@@ -187,6 +188,30 @@ def display_avatar(self) -> Asset:
         """
         return self.avatar or self.default_avatar
 
+    @property
+    def avatar_decoration(self) -> Optional[Asset]:
+        """Optional[:class:`Asset`]: Returns an :class:`Asset` for the avatar decoration the user has.
+
+        If the user has not set an avatar decoration, ``None`` is returned.
+
+        .. versionadded:: 2.4
+        """
+        if self._avatar_decoration_data is not None:
+            return Asset._from_avatar_decoration(self._state, self._avatar_decoration_data['asset'])
+        return None
+
+    @property
+    def avatar_decoration_sku_id(self) -> Optional[int]:
+        """Optional[:class:`int`]: Returns the SKU ID of the avatar decoration the user has.
+
+        If the user has not set an avatar decoration, ``None`` is returned.
+
+        .. versionadded:: 2.4
+        """
+        if self._avatar_decoration_data is not None:
+            return int(self._avatar_decoration_data['sku_id'])
+        return None
```

Comment:
```suggestion
        return _get_as_snowflake(self._avatar_decoration_data, 'sku_id')
```

Requires importing it above from `utils`.